webapp/channels/src/components/cloud_usage_modal/lhs_nearing_limit_modal.tsx

```diff
@@ -31,50 +30,50 @@ export default function LHSNearingLimitsModal() {
     const [limits] = useGetLimits();
 
     const primaryAction = {
-        message: {
-            id: t('workspace_limits.modals.view_plans'),
+        message: defineMessage({
```

Comment:
As I noted in another PR, we could group these together in one `defineMessages`, but using `defineMessage` here made the diff much easier to read